docs/keyless.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Keyless Mode
+description: Learn about Clerk's Keyless Mode.
+---
+
+**Keyless Mode** allows you to try out Clerk in your local development environment without having to sign-up for a Clerk account.
+
+## How does it work?
+
+If Clerk's SDK doesn't detect valid [Clerk keys](TODO) in your environment, it will automatically fetch a set of keys for you. These keys are associated with a Clerk development instance that has a default configuration. When running locally in Keyless Mode, you'll see a banner indicating that you're running in this special mode with a link to claim your instance by signing up for a Clerk account, or signing in if you already have one.
+
+Keys are written to a file in your project's `.clerk` directory. Ensure that this directory is added your `.gitignore` file to prevent keys from being committed to your repository.
```

Comment:
```suggestion
Keys are written to a file in a `.clerk` directory in your project. Ensure that this directory is added your `.gitignore` file to prevent keys from being committed to your repository.
```